gm4_blast_furnaces/data/gm4/advancements/blast_furnaces.json

```diff
@@ -0,0 +1,19 @@
+{
+  "display": {
+	"icon": {
+      "item": "minecraft:furnace",
+      "nbt":"{display:{Name:\"\\\"blast_furnaces logo\\\"\"}}"
+    },
+    "title":"Clever Smelting",
+    "description": {
+      "text": "Create your own Blast Furnace.",
+      "color":"gray"
+    }
+  },
+  "parent":"gm4:root",
+  "criteria": {
+    "blast_furnaces_trigger": {
```

Comment:
```suggestion
    "build_blast_furnace": {
```

Reply:
I'll change this. Haden't actually looked at this file yet.